Tiltfile

```diff
@@ -382,7 +382,7 @@ def enable_provider(name, debug):
 
 def find_object_name(objs, kind):
     for o in objs:
-        if o["kind"] == kind:
+        if o["kind"] == kind and "cluster.x-k8s.io/provider" in o["metadata"]["labels"]:
```

Comment:
This ensures the provider's `*_controller` item in tilt refers to the right Deployment.